exporter/awsxrayexporter/translator/segment.go

```diff
@@ -253,16 +253,14 @@ func convertToAmazonSpanID(v []byte) string {
 	return hex.EncodeToString(v[0:8])
 }
 
-func timestampToFloatSeconds(ts *timestamp.Timestamp, startTs *timestamp.Timestamp) float64 {
+func timestampToFloatSeconds(ts *timestamp.Timestamp) float64 {
 	var (
 		t time.Time
 	)
 	if ts == nil {
 		t = time.Now()
-	} else if startTs == nil {
-		t = time.Unix(ts.Seconds, int64(ts.Nanos))
 	} else {
-		t = time.Unix(startTs.Seconds, int64(ts.Nanos))
+		t = time.Unix(ts.Seconds, int64(ts.Nanos))
```

Comment:
Use this https://godoc.org/github.com/golang/protobuf/ptypes#Timestamp to convert?

Reply:
I made the change. Thanks for the suggestion